doc-content/enterprise-only/processes/generate-forms-proc.adoc

```diff
@@ -10,7 +10,7 @@ You have created the pizza orders business process. For more information, see <<
 .Procedure
 
 . Log in to {CENTRAL} and click *Menu* -> *Design* -> *Projects* -> *pizzaPlace* -> *pizzaOrders*.
-. Click the following menu option and select *Generate all forms*.
+. Select *Generate all forms* from the *Form Generation* pull-down menu.
```

Comment:
Consider deleting pull-down. If you do, search the doc for other instances.
IBMSG: pull down adj
Do not use. Use “pull-down,” but only if you must specify the type of menu or list.

Reply:
Done throughout. Thanks!